common/cards/default/hermits/geminitay-rare.ts

```diff
@@ -32,27 +32,32 @@ class GeminiTayRareHermitCard extends HermitCard {
 		const {player} = pos
 
 		// @TODO egg confusion, and how can we get rid of follow up
+		// The follow-up is now present for the counterpart.
 		// is that even in the scope of this refactor?
-		player.hooks.afterAttack.add(instance, (attack) => {
+		player.hooks.onAttack.add(instance, (attack) => {
 			if (attack.id !== this.getInstanceKey(instance) || attack.type !== 'secondary') return
 
-			// To keep this simple gem will discard the single use card, if it's used
-			if (player.board.singleUseCardUsed) {
+			player.hooks.afterAttack.add(instance, (attack) => {
+				// To keep this simple gem will discard the single use card, if it's used
+				// Hopefully, removing it doesn't break anything.
 				discardSingleUse(game, player)
-			}
+				player.board.singleUseCardUsed = false
```

Comment:
This is unnecessary, discardSingleUse does this

Reply:
Got it.